CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Updates should follow the [Keep a CHANGELOG](http://keepachangelog.com/) princip
 
 ## [Unreleased][unreleased]
 
+#### Added
+
+* PHP 8.1 support (`league/fractal` 0.20.1) (BC Break)
```

Comment:
It's not BC break as you still support old version. Maybe we should add all versions 17-21